client/my-sites/site-settings/search.jsx

```diff
@@ -147,92 +145,83 @@ class Search extends Component {
 		);
 	}
 
-	renderUpgradeNotice() {
-		return (
-			<Fragment>
-				<Banner
-					description={
-						this.props.siteIsJetpack
-							? this.props.translate(
-									'Jetpack Search replaces the built-in search with a fast, scalable, and customizable search powered by WordPress.com. The result: Your users find the content they want, faster.'
-							  )
-							: this.props.translate(
-									'The built-in WordPress search is great for sites without much content. But as your site grows, searches slow down and return less relevant results.'
-							  )
-					}
-					event={ 'calypso_jetpack_search_settings_upgrade_nudge' }
-					feature={ FEATURE_SEARCH }
-					plan={ this.props.siteIsJetpack ? PRODUCT_JETPACK_SEARCH : PLAN_BUSINESS }
-					title={
-						this.props.siteIsJetpack
-							? this.props.translate(
-									'Add faster, more advanced searching to your site with Jetpack Search'
-							  )
-							: this.props.translate(
-									'Add faster, more advanced searching to your site with WordPress.com Business'
-							  )
-					}
-				/>
-			</Fragment>
-		);
-	}
+	render() {
+		const {
+			siteId,
+			siteIsJetpack,
+			isSearchEligible,
+			searchModuleActive,
+			fields,
+			translate,
+			customizerUrl,
+		} = this.props;
+
+		// for now, don't even show upgrade nudge
+		if ( ! fields.jetpack_search_supported && ! isSearchEligible ) {
+			const upgradeTitle = siteIsJetpack
+				? translate( 'Add faster, more advanced searching to your site with Jetpack Professional' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Add faster, more advanced searching to your site with Jetpack Search' )` *ES Score: 11*